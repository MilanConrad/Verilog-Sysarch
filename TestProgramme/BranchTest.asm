lui $1 0xffff
bltz $1, jump
lui $1 2
jump:
lui $2 1
