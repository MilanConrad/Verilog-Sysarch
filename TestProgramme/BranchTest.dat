0x3c01ffff
0x04200001
0x3c010002
0x3c020001
