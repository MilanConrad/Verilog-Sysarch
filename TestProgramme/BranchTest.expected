@3 ffff0000
